### DISCUSSION TITLE
Getting Started..

### DISCUSSION THREAD

**Discussion by Nivas7** [cncf/mentoring#672] [Q&A] [answered]
Hey... I am new to open source and am going to get started give... some path to take contribution s and how it will work....

**Comment by nate-double-u** [comment, marked as answer]
Hi @Nivas7,

Thanks for your interest! If you want to be a Cloud Native contributor, I suggest starting by checking out the [CNCF Contributors](https://contribute.cncf.io/) page. A lot of the projects listed will have "good first issues" listed that are provided as a way to help newcomers get setup with smaller bits of work. Feel free to reach out to the communities too -- when you find an issue you think is interesting, ask questions!

**Reply by nate-double-u** [reply]
I should also suggest joining the [CNCF Slack](http://slack.cncf.io/) if you haven't yet, a lot of projects use it as a communication hub.
